disnake/interactions/modal.py

```diff
@@ -134,19 +142,18 @@ class ModalInteractionData(Dict[str, Any]):
     ----------
     custom_id: :class:`str`
         The custom ID of the modal.
+    components: List[:class:`dict`]
+        The raw component data of the modal interaction.
+
+        .. versionadded:: 2.6
```

Comment:
Should probably add something like 
> ...as provided by Discord. This is incomplete due to limitations by Discord.

I feel like the current wording people will miss that its incomplete

Reply:
Good point, added in acca2ae01f712587e6d0c111b868801e49502c60.